packages/.eslintrc.json

```diff
@@ -1,14 +1,9 @@
 {
```

Comment:
plz use latest version for this file.

https://github.com/DimensionDev/Maskbook/blob/develop/packages/.eslintrc.json

Reply:
done